src/maker/api.rs

```diff
@@ -251,6 +268,8 @@ impl Maker {
             config.connection_type = connection_type;
         }
 
+        let thread_pool_port = config.port;
```

Comment:
remove this variable. directly call `config.port` at call site.

Reply:
The `port` value is moved and it was a borrow-checker issue, since the `config.port` is being used after `config` has been moved into the `Maker` struct in initial fields.

Reply:
Okay makes sense. Cool then.